frontend/www/js/service-worker.js

```diff
@@ -30,25 +28,61 @@ self.addEventListener('push', async function(event) {
     });
 });
 
-buerro_url = 'http://localhost:4000/'
+/*
```

Comment:
What is this comment for ?

Reply:
Was a feature idea: should notifications be pushed if the user has the window opened and focused?

Reply:
You mean if the buerro should create notifications if the user is already in the buerro window ? - > I think it would be more convenient for the user if the notification is displayed in the window itself

Reply:
A new message will appear anyway in most cases.

Reply:
I think of it, I would prefer it to be that way.

If the window is currently focused there shouldn't be a notification, just the message should be enough.

Reply:
yes, I agree

Reply:
Shall we merge then ?

Reply:
No, I will apply that change